### DISCUSSION TITLE
I really want to print a degree character but all I get is Chinese

### DISCUSSION THREAD

**Discussion by CzarAlex** [python-escpos/python-escpos#504] [Q&A] [answered]
chr 176 is what I'm looking for so I can display temperature degrees. Is there an easy fix for this?

**Comment by belono** [comment, marked as answer]
Hi @CzarAlex .

Maybe [this](https://github.com/python-escpos/python-escpos/issues/37#issuecomment-1368844563) comment can help you.

Note that your printer must contain that character in the selected codepage. Otherwise, the only way is to render the text as an image and send it through `.image()`
